### DISCUSSION TITLE
Add "Name Translations" for "Repeater" Interface Fields

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#12897] [Feature Requests]
- In a collection you can use "Collection Naming Translations".
- In a field you can use "Field Name Translations".
- **In a "Repeater" interface there is no equivalent.**

This is quite odd considering you can add translations for the "Note" option on the repeater field. And if the field is a string input, you can add translations for the "Placeholder" option. Yet you cannot add translations for the field name itself.

**Comment by rijkvanzanten** [comment]
It's because the `field` in there is the JSON key that's used in the stored value, not just the display name 👍🏻 

I think the solution here is to add a dedicated "field name" option that can then use the translated key 🙂

**Comment by henk23** [comment]
This is already kind of possible when you have access to the database fields.

Go to the `directus_fields` table and look into the `options` column for you repeater field.
The `addLabel` for the whole repeater field as well as the `name` of the single fields can be translation strings.

```json
{
  "addLabel": "$t:my_custom_addLabel_translation",
  "fields": [
    {
      "field": "firstname",
      "name": "$t:my_custom_firstname_translation",
      "type": "string",
      "meta": {...}
    },
    {
      "field": "lastname",
      "name": "$t:my_custom_firstname_translation",
      "type": "string",
      "meta": {...}
    }
  ],
  "template": "{{ firstname }} {{ lastname }}"
}
```

This is just not possible in the Directus App yet, this functionality should be added.

**Comment by AaronBeaudoin** [comment]
Just ran into this issue again. I searched the discussions here and then found my own issue from early last year. It's great that it is possible to use a translation string via direct database access, but in my setup database access is extremely inconvenient, so being able to do this via the UI would be _so much better_.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by svardys** [reply]
I'm really sorry about this.  
This functionality would really help me a lot.
